### PR TITLE
Airspeed Filtering

### PR DESCRIPTION
**Pull request by dewagter**

### CODE REVIEW

File: sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c [hunk 1]
```diff
@@ -221,13 +223,22 @@ void ms45xx_i2c_init(void)
...
 {
+#ifdef USE_AIRSPEED_LOWPASS_FILTER
+  // Update the filter if needed
+  if(ms45xx.lowpass_tau != ms45xx.lowpass_tau_set) {
+    ms45xx.lowpass_tau_set = ms45xx.lowpass_tau;
+    init_butterworth_2_low_pass(&ms45xx_filter, ms45xx.lowpass_tau,
+                              MS45XX_I2C_PERIODIC_PERIOD, get_butterworth_2_low_pass(&ms45xx_filter));
+  }
+#endif
+
```
Comment (gautierhattenberger) [inline sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c:234]: maybe using a setting handler is a better way to update the time constant and call the init with the newly received value ?

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

File: sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c (hunk 1 above)
Comment (dewagter) [reply sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c:234]: Do we have the guarantee that the telemetry callback and i2c_periodic are not executed in parallel in chibios?

Comment (gautierhattenberger) [reply sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c:234]: They are both part of the mainloop, only the low level transactions are using dedicated threads (accessed through protected buffers and transaction queues)

**Review by gautierhattenberger — APPROVED** [review]

File: sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c (hunk 1 above)
Comment (dewagter) [reply sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c]: Updated to use a handler

**Pull request merged**